providers/os/id/clouddetect/clouddetect.go

```diff
@@ -29,16 +29,25 @@ var detectors = []detectorFunc{
 	gcp.Detect,
 }
 
+const AssetKind = "virtualmachine"
```

Comment:
Lets find a better name for this

Reply:
The shared kinds probably belong into the provider sdk

Reply:
I can't put it there since we will have a cyclical dependency, but I think it would be a great idea to move these constants (for the near future) into the actual `inventory` package.

I think we will benefit from using them in the backend.